data/oftenlinks.json

```
@@ -43,6 +43,11 @@
   "description": "Das zentrale Mailsystem der HAW Hamburg",
   "keywords": [],
   "url": "https://haw-mailer.haw-hamburg.de/owa"
+}, {
+  "name": "HAW VPN Service",
+  "description": "VPN Server der HAW Hamburg",
+  "keywords": ["vpn"],
```

I think more keywords might be helpul
```suggestion
  "keywords": ["vpn", "cisco", "openconnect", "paper", "research", "icc"],
```